pyproject.toml

```diff
@@ -53,7 +53,7 @@ nest-asyncio = "^1.5.8"
 nltk = "^3.8.1"
 numpy = "*"
 openai = ">=1.1.0"
-pandas = {extras = ["jinja2"], version = "*"}
+pandas = {version = "*"}
```

Comment:
Technically this could be `pandas = "*"`

Reply:
Nice suggestion.  Will move to that